data/en/test_radio_mandatory_with_mandatory_other.json

```diff
@@ -0,0 +1,48 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.2",
+    "survey_id": "0",
+    "title": "Other input fields",
+    "theme": "default",
+    "legal_basis": "StatisticsOfTradeAct",
+    "groups": [{
+        "id": "radio",
+        "title": "RadioMandatory",
```

Comment:
The title should reflect the survey - this should be something like "Mandatory Radio with Mandatory Other". This applies to all the other schemas.